blazor/daterangepicker/how-to/open-popup-on-input-click.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Open the Blazor DateRangePicker popup on Focus | Syncfusion
+description: Learn here all about opening the Syncfusion Blazor DateRangePicker popup upon focusing input and much more.
+platform: Blazor
+control: DateRangePicker
+documentation: ug
+---
+
+# Open the Blazor DateRangePicker popup on Focus
+
+You can also open the DateRangePicker popup on input focus by setting the `OpenOnFocus` property to true.
+
+The following example demonstrates how to open the DateRangePicker popup when the input is focused.
+
+```cshtml
+@using Syncfusion.Blazor.Calendars
+
+<SfDateRangePicker TValue="DateTime?" Placeholder="Choose a range" OpenOnFocus="true" ShowClearButton="true"></SfDateRangePicker>
+
+```
+
+![Opening Blazor DateRangePicker Popup](../images/blazor-daterangepicker-open-focus.gif)
```

Comment:
Set a proper width for the component in the GIf

Reply:
ok sir